pkg/cloudprovider/yandex/cloud.go

```diff
@@ -69,15 +82,20 @@ func NewCloudConfig() (*CloudConfig, error) {
 		var err error
 		folderID, err = metadata.GetFolderID()
 		if err != nil {
-			return nil, errors.Wrapf(err, "cannot get FolderID from instance metadata")
+			return nil, errors.Wrap(err, "cannot get FolderID from instance metadata")
 		}
 	}
 	cloudConfig.FolderID = folderID
 
-	// Retrieve LocalZone
-	localZone, err := metadata.GetZone()
-	if err != nil {
-		return nil, errors.Wrap(err, "cannot get Zone from instance metadata")
+	// Retrieve FolderID
+	// firstly - try to find it in env. variables
+	localZone := os.Getenv(envZone)
```

Comment:
Is there a point to overriding zone?

Reply:
For local development purposes but I will move this to another pr